src/bin/compute-wer-bootci.cc

```diff
@@ -116,9 +116,9 @@ void GetBootstrapWERInterval(
       BaseFloat *mean, BaseFloat *interval) {
     BaseFloat wer_accum = 0.0, wer_mult_accum = 0.0;
 
-    for (int32 i = 0; i <= replications; ++i) {
+    for (int32 i = 0; i < replications; ++i) {
       int32 num_words = 0, word_errs = 0;
-      for (int32 j = 0; j <= edit_word_per_hyp.size(); ++j) {
+      for (int32 j = 0; j < edit_word_per_hyp.size(); ++j) {
         int32 random_pos = kaldi::RandInt(0, edit_word_per_hyp.size());
```

Comment:
RandInt gives you a random integer between a and b inclusive.  So I suspect there should be a -1 here.